### PR TITLE
minor changes to aex-2 integration

### PR DESCRIPTION
**Pull request by kenodressel**

### CODE REVIEW

File: src/utils/walletSearch.js [hunk 1]
```diff
@@ -40,9 +40,11 @@ export const wallet = {
     const handleWallets = async function ({ wallets, newWallet }) {
       detector.stopScan();
       const connected = await this.client.connectToWallet(await newWallet.getConnection());
-      this.client.selectNode(connected.networkId); // connected.networkId needs to be defined as node in RpcAepp
+      // TODO re-enable once fixed
```
Comment (thepiwo) [inline src/utils/walletSearch.js]: What's not working with this, was working well in my testing

Comment (kenodressel) [reply src/utils/walletSearch.js]: it works well, if the wallet has the most up-to-date sdk, but without this it works in the superhero waellet as is.

Comment (thepiwo) [reply src/utils/walletSearch.js]: I think we should only target most up-to-date sdk anyways, let's ask the superhero wallet to be updated

**Review by thepiwo — APPROVED** [review]
lgtm

**Pull request merged**